spec/system/candidate_interface/entering_details/candidate_entering_other_qualification_no_qualification_option_choice_spec.rb

```diff
@@ -109,7 +109,7 @@ def then_i_see_the_other_qualification_review_page
   end
 
   def and_i_dont_see_my_no_other_qualification_selected
-    expect(page).not_to have_content('A levels and other qualifications')
+    expect(page).not_to have_content('I do not want to add any A levels and other qualifications')
```

Comment:
From what I understood from the PR is that the assertion needed. Let me know if is different.

Reply:
Perfect, thanks!